homeassistant/components/camera/__init__.py

```diff
@@ -684,7 +685,7 @@ async def async_handle_play_stream_service(camera, service_call):
     )
     data = {
         ATTR_ENTITY_ID: entity_ids,
-        ATTR_MEDIA_CONTENT_ID: f"{hass.config.api.base_url}{url}",
+        ATTR_MEDIA_CONTENT_ID: f"{async_get_url(hass, require_ssl=True)}{url}",
```

Comment:
I don't think that SSL is necessary. This is just sending urls to media players.